lutris/gui/lutriswindow.py

```diff
@@ -996,28 +996,39 @@ def on_sidebar_changed(self, widget):
 
     @watch_errors()
     def on_game_selection_changed(self, view, selection):
+        game_ids = []
+        games = []
         if not selection:
             GLib.idle_add(self.update_revealer)
             return False
-        game_id = view.get_model().get_value(selection, COL_ID)
-        if not game_id:
+        if type(selection) != list:
+            game_id = view.get_model().get_value(selection, COL_ID)
```

Comment:
This is over complicated; just use a list here *all the time*. A one element list is perfectly fine when only one game is selected.

Reply:
Turning to only list type is ok for the grid view, but list view returns a single treeiter object, not a list.

Reply:
This needs to be consistent. Both versions should generate lists, perhaps lists of iterators or of game-ids. But the same thing whether it is a list or a grid.

Reply:
Ok. Will need to work on that